docs/cmd/kn_channel_describe.md

```diff
@@ -0,0 +1,43 @@
+## kn channel describe
+
+Show details of a channel
+
+### Synopsis
+
+Show details of a channel
+
+```
+kn channel describe NAME
+```
+
+### Examples
+
+```
+
```

Comment:
new line

Reply:
thats auto-generated by cobra and we've been following this new line format for all the resource CRUD
see for eg https://github.com/knative/client/blob/master/docs/cmd/kn_broker_create.md